releng/lsp4j-feature/feature.xml

```diff
@@ -31,7 +31,7 @@
    </license>
 
    <requires>
-      <import plugin="com.google.gson" version="2.8.9" match="equivalent"/>
+      <import plugin="com.google.gson" version="2.8.9" match="compatible"/>
```

Comment:
i am not sure if we just should go to 2.9.0 as minimal version.

Reply:
@jonahgraham what do you think?

Reply:
Lets make 2.9.0 min version.

Reply:
done